conf/protists/allowed_species.json

```
@@ -0,0 +1,138 @@
+    "naegleria_gruberi_gca_000004985",
+    "nannochloropsis_gaditana_ccmp526_gca_000240725",
+    "nothophytophthora_sp_chile5_gca_001712635",
+    "oxytricha_trifallax_gca_000711775",
+    "paramecium_tetraurelia",
+    "perkinsela_sp_ccap_1560_4_gca_001235845",
+    "perkinsus_marinus_atcc_50983_gca_000006405",
+    "peronospora_effusa_gca_003704535",
+    "phaeodactylum_tricornutum",
+    "phytomonas_sp_isolate_em1_gca_000582765",
+    "phytomonas_sp_isolate_hart1_gca_000982615",
+    "phytophthora_cactorum_gca_003287315",
+    "phytophthora_infestans",
+    "phytophthora_kernoviae",
+    "phytophthora_lateralis",
+    "phytophthora_megakarya_gca_002215365",
+    "phytophthora_nicotianae_gca_001482985",
+    "phytophthora_palmivora_var_palmivora_gca_002911725",
+    "phytophthora_parasitica",
+    "phytophthora_ramorum",
+    "phytophthora_sojae",
+    "planoprotostelium_fungivorum_gca_003024175",
+    "plasmodiophora_brassicae_gca_001049375",
+    "plasmodium_berghei",
+    "plasmodium_chabaudi",
+    "plasmodium_coatneyi_gca_001680005",
+    "plasmodium_cynomolgi_strain_b_gca_000321355",
+    "plasmodium_falciparum",
+    "plasmodium_fragile_gca_000956335",
+    "plasmodium_gaboni_gca_001602025",
+    "plasmodium_gallinaceum_gca_900005855",
+    "plasmodium_gonderi_gca_002157705",
+    "plasmodium_inui_san_antonio_1_gca_000524495",
+    "plasmodium_knowlesi",
+    "plasmodium_malariae_gca_900088575",
+    "plasmodium_ovale_curtisi_gca_900088565",
+    "plasmodium_reichenowi_gca_001601855",
+    "plasmodium_relictum_gca_900005765",
+    "plasmodium_sp_drc_itaito_gca_900240055",
+    "plasmodium_sp_gorilla_clade_g2_gca_900097015",
+    "plasmodium_vinckei_petteri_gca_000524515",
+    "plasmodium_vivax",
+    "plasmodium_yoelii_yoelii_gca_000003085",
+    "plasmopara_halstedii_gca_900000015",
+    "pseudocohnilembus_persalinus_gca_001447515",
+    "pseudonitzschia_multistriata",
+    "pythium_aphanidermatum",
+    "pythium_arrhenomanes",
+    "pythium_irregulare",
+    "pythium_iwayamai",
+    "pythium_ultimum",
+    "pythium_vexans",
+    "reticulomyxa_filosa_gca_000512085",
+    "salpingoeca_rosetta_gca_000188695",
+    "saprolegnia_diclina_vs20_gca_000281045",
+    "saprolegnia_parasitica_cbs_223_65_gca_000151545",
+    "sphaeroforma_arctica_jp610_gca_001186125",
+    "spironucleus_salmonicida_gca_000497125",
+    "stentor_coeruleus_gca_001970955",
+    "strigomonas_culicis_gca_000442495",
+    "stylonychia_lemnae_gca_000751175",
+    "symbiodinium_microadriaticum_gca_001939145",
+    "tetrahymena_thermophila",
+    "thalassiosira_oceanica_gca_000296195",
+    "thalassiosira_pseudonana",
+    "thecamonas_trahens_atcc_50062_gca_000142905",
+    "theileria_equi_strain_wa_gca_000342415",
+    "theileria_orientalis_strain_shintoku_gca_000740895",
+    "theileria_parva_gca_000165365",
+    "thraustotheca_clavata_gca_002081575",
+    "tieghemostelium_lacteum_gca_001606155",
+    "toxoplasma_gondii_veg_gca_000150015",
+    "toxoplasma_gondii",
```

A suggestion as optional as it is pedantic:
```suggestion
    "toxoplasma_gondii",
    "toxoplasma_gondii_veg_gca_000150015",
```